payments/trustly/index.md

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Could not find module `remark-slug`](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5607142)